### ISSUE TITLE
Floating bug: In file type `html` inside `script` tags doesnt work one line comment command until your open/activate another tab with `.js` file extension

### ISSUE THREAD

**Issue by psnet** [microsoft/vscode#235344]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

1. ALL vs code IDE windows must be closed
2. Open IDE, it must be at startup open previous project at tab with `html` file with input cursor put inside `script` tag
3. Try to use comment line command (`ctrl + /`) to comment JS code inside this `script` tag

AR
nothing happens (until your click on another tab with `.js` file)

ER
must be commented line

This happens from time to time, but not always

**Comment by aeschli** [comment]
That works for me.

What's the editor language mode (you can see this in the status bar). HTML?

**Comment by psnet** [comment]
Yes, `html`. This works if you open first window IDE and it opens last project and cursor already stay inside `script` tag (anywhere). It works in 30% of time. It feels like that JS handler (or whatever controls js code inside `script` tags) is not loaded (and not loading, waiting minute or two doesnt fixes this)

**Comment by aeschli** [comment]
Ok, I see, if you reload the Window and press `ctrl + /` right away, then the htm comment is used.

I suspect the TextMate grammar or the embedded grammar are not yet loaded.

The 'Add Line Comment` command should await these

**Comment by RedCMD** [comment]
make sure to reload VSCode with the embedded language (JavaScript) not visible on the screen

often the embedded languages are not correctly loaded on reload
and no amount of time waiting for it fixes it
however, often typing fixes it

https://github.com/user-attachments/assets/1c7beb39-bd84-43d4-9add-101e496d01da

https://github.com/user-attachments/assets/e70e54a0-7689-4b9a-a212-31b97b5fdfb7


related:
* https://github.com/microsoft/vscode/issues/235241
* https://github.com/microsoft/vscode/issues/234867
* https://github.com/microsoft/vscode/issues/234722
* https://github.com/microsoft/vscode/issues/234622
* https://github.com/microsoft/vscode/issues/231168
* https://github.com/microsoft/vscode/issues/216587
* https://github.com/microsoft/vscode/issues/210928
* https://github.com/microsoft/vscode/issues/210888
* https://github.com/microsoft/vscode/issues/208281
* https://github.com/microsoft/vscode/issues/201702
* https://github.com/microsoft/vscode/issues/133462

**Comment by RedCMD** [comment]
also @aeschli 
@psnet stated that comment wasn't working completely 
not just that html comments were working instaed

**Comment by RedCMD** [comment]
this is not actually a dup of https://github.com/microsoft/vscode/issues/202938
even tho the symptoms are exactly the same
this was working correctly in 1.92.2
but broke in 1.93.0

caused by one of this commits https://github.com/microsoft/vscode/compare/36e4ddb4dbdf5d41bbeedfce2db4834085de2b87...e81aceaa07bab1ccdc94deae04168c8bd062f0de

**Comment by RedCMD** [comment]
@hediet 
caused by commit https://github.com/microsoft/vscode/commit/29be8e8665989b2b5c217ea7f7ce5df3d90c9d33
* https://github.com/microsoft/vscode/pull/226410
* https://github.com/microsoft/vscode/issues/225794
